### PR TITLE
feat: store event tags

### PR DESCRIPTION
**Pull request by pvlugter**
Refs #30

### CODE REVIEW

File: core/src/main/scala/akka/persistence/dynamodb/internal/QueryDao.scala [hunk 1]
```diff
@@ -89,7 +89,7 @@ import software.amazon.awssdk.services.dynamodb.model.QueryRequest
             serId = item.get(EventSerId).n().toInt,
             serManifest = item.get(EventSerManifest).s(),
             writerUuid = item.get(Writer).s(),
-            tags = Set.empty, // FIXME
+            tags = if (item.containsKey(Tags)) item.get(Tags).ss().asScala.toSet else Set.empty,
```
Comment (patriknw) [inline core/src/main/scala/akka/persistence/dynamodb/internal/QueryDao.scala:92]: ah, I was looking for such containsKey in some other place, but used `get` and null check. good to know

File: core/src/test/scala/akka/persistence/dynamodb/query/EventsBySliceSpec.scala [hunk 2]
```diff
@@ -255,8 +255,26 @@ class EventsBySliceSpec
...
+            .runWith(TestSink())
+
+        result.request(3)
+        val envelopes = result.expectNextN(3)
+        envelopes.map(_.tags) should ===(Seq(Set("tag-A"), Set("tag-A"), Set("tag-A")))
+
+        query.loadEnvelope[String](persistenceId.id, 1L).futureValue.tags shouldBe Set("tag-A")
+
+        assertFinished(result)
+      }
```
Comment (patriknw) [inline core/src/test/scala/akka/persistence/dynamodb/query/EventsBySliceSpec.scala:277]: could also port `PersistTagsSpec`, but maybe it's not adding more coverage? The original reason for that test could have been that first we only stored the tags, but didn't read them with the queries.

**Review by patriknw — APPROVED** [review]
LGTM

File: core/src/test/scala/akka/persistence/dynamodb/query/EventsBySliceSpec.scala (hunk 2 above)
Comment (pvlugter) [reply core/src/test/scala/akka/persistence/dynamodb/query/EventsBySliceSpec.scala:277]: Only saw the FIXME for this test. Not sure if `PersistTagsSpec` adds much. Could be useful to test tags for snapshots, when adding projection start from snapshot?

**Pull request merged**